csharp/src/Google.Protobuf.Test/project.json

```diff
@@ -27,6 +27,7 @@
   "testRunner": "nunit",
 
   "frameworks": {
+    "net451": {},
```

Comment:
Yes, but I seem to remember there was some reason why I needed to make the tests depend on net451. I can't remember what offhand, but I'll have a look and change it if I can.